app/views/static_pages/contact.html.erb

```diff
@@ -1,4 +1,5 @@
 <% provide(:title, "Contact") %>
+<p></p>
```

Comment:
@shaniajubelag Same comment in https://github.com/framgia/ph_sels_rails_shania/pull/8/files#diff-39ce76d94a7eb2d6282c0ff8a4ce44a9R2